frontend/src/app/main/data/style_dictionary.cljs

```diff
@@ -352,19 +352,17 @@
         (cond
           (nil? tokens) nil
           ;; The tokens are already processing somewhere
-          (p/promise? cached) (-> cached
-                                  (p/then #(reset! tokens-state %))
-                                  #_(p/catch js/console.error))
+          (rx/observable? cached) (rx/sub! cached #(reset! tokens-state %))
           ;; Get the cached entry
           (some? cached) (reset! tokens-state cached)
           ;; No cached entry, start processing
-          :else (let [promise+ (if interactive?
-                                 (resolve-tokens-interactive+ tokens)
-                                 (resolve-tokens+ tokens))]
-                  (swap! cache-atom assoc tokens promise+)
-                  (p/then promise+ (fn [resolved-tokens]
-                                     (swap! cache-atom assoc tokens resolved-tokens)
-                                     (reset! tokens-state resolved-tokens)))))))
+          :else (let [resolved-tokens (if interactive?
```

Comment:
When a variable contains not a final value but a rx stream, we usually add the `-s` suffix. For example `resolved-tokens-s` in this case. Same applies to other variables below.